test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_equals_null.test

```diff
@@ -29,8 +29,6 @@ brokerStart CB 0-255
 
 --SHELL--
 
-#
-# FIXME P2, once 'null' values are supported (issue #2359), this test should be enabled
```

Comment:
It seems we fixed #2359 but forget to enable this .test... :)

NTC (informative)

Reply:
In addition, a pararell test for mq= has been added as new .test file